.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,22 @@
+name: 'Chromatic'
+
+on: [push, workflow_dispatch]
+
```

Comment:
Do we want this limited to the `main` branch too?

Reply:
No, this allows it to run on PR's for visual change approvals

Reply:
see https://www.chromatic.com/docs/github-actions#recommended-configuration-for-build-events